.gitmodules

```diff
@@ -22,3 +22,6 @@
 [submodule "thirdparty/yaml-cpp"]
 	path = thirdparty/yaml-cpp
 	url = https://github.com/jbeder/yaml-cpp.git
+[submodule "thirdparty/substrait-cpp"]
```

Comment:
why still need submodule for substrait-cpp, you can git clone the code with externalproject

Reply:
Done.